src/app/components/Video/index.jsx

```diff
@@ -1,51 +1,38 @@
 import React from 'react';
-import { videoComponentPropTypes } from '../../models/propTypes';
+import { shape } from 'prop-types';
+import deepGet from '../../helpers/json/deepGet';
 
-const Video = ({
-  videoLocator,
-  duration,
-  rawImageSrc,
-  versionID,
-  imageLocator,
-  kind,
-}) => (
-  <div>
-    <div>
-      video locator:
-      {videoLocator}
-    </div>
-    <div>
-      duration:
-      {duration}
-    </div>
-    <div>
-      raw Image Src:
-      {rawImageSrc}
-    </div>
-    <div>
-      versionID:
-      {versionID}
-    </div>
-    <div>
-      imageLocator:
-      {imageLocator}
-    </div>
+const Video = ({ model }) => {
+  const nestedModel = deepGet(['blocks', 0, 'model'], model);
```

Comment:
Looks good so far!

Moving this data fetching to the Video Container would be better. We aim to have a separation - Components being presentational and only having the necessary props passed in. 
Containers - including the data manipulation necessary and passing props to the Component.

Reply:
Thanks, recent commit reflects this suggestion 🙂